pkg/apis/batch/v1alpha1/job.go

```diff
@@ -70,6 +70,15 @@ type JobSpec struct {
 	// Defaults to 3.
 	// +optional
 	MaxRetry int32 `json:"maxRetry,omitempty" protobuf:"bytes,8,opt,name=maxRetry"`
+
+	// ttlSecondsAfterFinished limits the lifetime of a Job that has finished
+	// execution (either Completed or Failed). If this field is set,
+	// ttlSecondsAfterFinished after the Job finishes, it is eligible to be
+	// automatically deleted. If this field is unset,
+	// the Job won't be automatically deleted. If this field is set to zero,
+	// the Job becomes eligible to be deleted immediately after it finishes.
+	// +optional
+	TTLSecondsAfterFinished *int32 `json:"ttlSecondsAfterFinished,omitempty" protobuf:"varint,8,opt,name=ttlSecondsAfterFinished"`
```

Comment:
Should we set a default value for this ?  Current behavior is when no set, gc will not delete Job.

Reply:
index -> 9

Reply:
> Should we set a default value for this ?  Current behavior is when no set, gc will not delete Job.

Yes, we should set default for this in admission controller.

Reply:
> index -> 9

Will update kube-batch